core/tests/protractor_desktop/contributorDashboard.js

```diff
@@ -45,7 +45,7 @@ describe('Contributor dashboard page', function() {
   const USER_EMAILS = ['user0@contributor.com', 'user1@contributor.com'];
   const QUESTION_ADMIN_EMAIL = 'user@contributor.com';
   const QUESTION_ADMIN_USERNAME = 'user4321';
-  const HINDI_LANGUAGE = 'Hindi';
+  const GERMAN_LANGUAGE = 'Deutsch (German)';
```

Comment:
It was getting difficult to match for characters which are not in english resulting in flakiness, so changed the language to German!